static/app/utils/fields/index.ts

```diff
@@ -1382,6 +1386,20 @@ const REPLAY_CLICK_FIELD_DEFINITIONS: Record<ReplayClickFieldKey, FieldDefinitio
     kind: FieldKind.FIELD,
     valueType: FieldValueType.STRING,
   },
+  [ReplayClickFieldKey.DEAD_SELECTOR]: {
+    desc: t(
+      'query using CSS selector-like syntax, supports class, id, and attribute selectors'
```

Comment:
Should the description describe what you're querying for?

Reply:
ah I copied the existing description from `CLICK_SELECTOR`